### ISSUE TITLE
Parsing

### ISSUE THREAD

**Issue by norf315** [sande807/4061projects#2]
In main.c within the parse function we need to read each line and decide whether the incoming line is giving us a target, a dependency or a command. could have no commands. could have no dependencies. will always have a target name and include a colon after. Each target needs to create its own linked list structure, keep track of its command and point to it's dependencies.

**Comment by sande807** [comment]
worked on parsing function a bit... i outlined what needs to be done and unsuccessfully played around with it. we still need to create a 'target' object

**Comment by norf315** [comment]
ok so I figured out some parsing things, the key is to compare the first space of the pointer (lpszLine[0]) to a character like '\t' for tab or '#' for comments. The problem is it looks like were suppose to automatically determine that the line after a target is the command, and throw the command out if it doesnt begin with a '\t'.

**Comment by norf315** [comment]
the parsing is beginning to behave as the TA discussed that it should on the forum post about whitespace. I got it to ignore comments between the target/dependency line and the command line which begins with a tab. I made changes to the makefile to simulate these problems. run make4061 if you want to see it print stuff out. If it looks good then the next step should be to put the target, dependencies, and command in the target structure.
